### PR TITLE
[15.0][MIG] survey_resource_booking: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT41024

@chienandalu @ernestotejeda please review!

### CODE REVIEW

File: survey_resource_booking/tests/test_invite.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        with self.assertRaises(UserError):
+            booking.action_invite_survey()
+        # Bad mail? Nope
+        self.partner.email = "I'm tricking you"
+        with self.assertRaises(UserError):
+            booking.action_invite_survey()
+        # After setting good mail, yep
+        self.partner.email = "donkey@banana.kong"
+        # import wdb;
+        # wdb.set_trace()
```
Comment (ernestotejeda) [inline survey_resource_booking/tests/test_invite.py]:
```suggestion
```

**Review by ernestotejeda — CHANGES_REQUESTED** [review]

File: survey_resource_booking/tests/test_invite.py (hunk 1 above)
Comment (stefan-tecnativa) [reply survey_resource_booking/tests/test_invite.py]: Ups, I forgot that. Thanks.

**Review by ernestotejeda — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**